src/HistoryService.Data/Interfaces/IServiceRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using LT.DigitalOffice.HistoryService.Models.Db;
+using LT.DigitalOffice.Kernel.Attributes;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.HistoryService.Data.Interfaces
+{
+    [AutoInject]
+    public interface IServiceRepository
+    {
+        Guid? Create(DbService dbService);
+
+        bool DoesServiceNameExist(string name);
```

Comment:
you in service repo, i think no need to write what it is service method

Reply:
done